JS/popup-content.js

```diff
@@ -0,0 +1,178 @@
+    content: `Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+    At eos qui possimus nisi ducimus delectus enim beatae natus 
+    ab? Natus minima minus similique nisi eos, deleniti laudantium 
+    nihil impedit assumenda! ducimus delectus enim beatae natus 
+    ab? Natus minima minus`,
+    languages: ['HTML', 'css', 'javaScript'],
+  },
+  {
+    name: 'Tonic',
+    work: ['CANOPY', 'Back End Dev', '2015'],
+    content: `Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+    At eos qui possimus nisi ducimus delectus enim beatae natus 
+    ab? Natus minima minus similique nisi eos, deleniti laudantium 
+    nihil impedit assumenda! ducimus delectus enim beatae natus 
+    ab? Natus minima minus`,
+    languages: ['HTML', 'css', 'javaScript'],
+  },
+  {
+    name: 'Tonic',
+    work: ['CANOPY', 'Back End Dev', '2015'],
+    content: `Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+    At eos qui possimus nisi ducimus delectus enim beatae natus 
+    ab? Natus minima minus similique nisi eos, deleniti laudantium 
+    nihil impedit assumenda! ducimus delectus enim beatae natus 
+    ab? Natus minima minus`,
+    languages: ['HTML', 'css', 'javaScript'],
+  },
+  {
+    name: 'Multi-Post Stories',
+    work: ['CANOPY', 'Back End Dev', '2015'],
+    content: `Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+    At eos qui possimus nisi ducimus delectus enim beatae natus 
+    ab? Natus minima minus similique nisi eos, deleniti laudantium 
+    nihil impedit assumenda! ducimus delectus enim beatae natus 
+    ab? Natus minima minus`,
+    languages: ['HTML', 'css', 'javaScript'],
+  },
+];
+
+const modal0 = document.querySelector('.modal0');
+
+modal0.insertAdjacentHTML(
+  'afterbegin',
+  `
+    <div class='modal-header'>
+        <h2 class='title'>${allContent[0].name}</h2>
+        <button data-close-button type='button' class='close-button'>&times;</button>
+    </div>
+    <ul class='sub-title'>
+        <li>${allContent[0].work[0]}</li>
+        <li>${allContent[0].work[1]}</li>
+        <li>${allContent[0].work[2]}</li>
+    </ul>
+    <div class='modal-photo' id='photo11'>
+    </div>
+    <section class='modal-body'>
+        <p class='modal-body1'>${allContent[0].content}</p>
+        <div class='modal-body2'>
+            <ul class='category'>
+                <li><a href='index.html'>${allContent[0].languages[0]}</a></li>
+                <li><a href='index.html'>${allContent[0].languages[1]}</a></li>
+                <li><a href='index.html'>${allContent[0].languages[2]}</a></li>
+            </ul>
+            <div class='extra-buttons'>  
+            <button class='project'><p>See Live</p><img src='image/sm1.png'></button>
+            <button class='project'><p>See Source</p><img src='image/sm2.png'></button>
+            </div>
+        </div>
+    </section> 
+
+`,
+);
+
+const modal1 = document.querySelector('.modal1');
+
+modal1.insertAdjacentHTML(
```

Comment:
- [ ] Kindly make sure to stick to the design, ensuring your design doesn't breack as long as we are between 375 - 1440 px of width.

Your breaking desings:
![Screenshot from 2023-02-11 15-34-34](https://user-images.githubusercontent.com/97834160/218260880-5d1f12e0-c75a-4505-b80e-d9a6d6ceccd5.png)
![Screenshot from 2023-02-11 15-35-09](https://user-images.githubusercontent.com/97834160/218260924-ccc8d6da-f224-43a0-9d2c-4fc70f6f1bbe.png)

- [ ] I think it would be better to make sure the width of your popup occupies more space as shown on the figma design below and also as you can see the popup it self is not scollable, Kindly set the to overlay instead

![Screenshot from 2023-02-11 15-41-06](https://user-images.githubusercontent.com/97834160/218261153-5bcb4d67-7867-48a5-9470-4d54451ea5ef.png)